lib/matplotlib/figure.pyi

```diff
@@ -11,7 +11,7 @@ from matplotlib.backend_bases import (
 from matplotlib.colors import Colormap, Normalize
 from matplotlib.colorbar import Colorbar
 from matplotlib.cm import ScalarMappable
-from matplotlib.gridspec import GridSpec, SubplotSpec
+from matplotlib.gridspec import GridSpec, SubplotSpec, SubplotParams
```

Comment:
If you want this to be re-exported in the type stub, it needs to be `SubplotParams as SubplotParams`.